gstack-buffer/alloca/README.md

```diff
@@ -0,0 +1,8 @@
+### What is alloca?
```

Comment:
add info how, when and why we should compile

Reply:
We don't have to compile it. On wasm32-unknown-unknown we use pre-built library so that the end user of gstd does not need installed C compiler when building contracts. However, the end user will benefit from our optimizations due to the pre-built library.

Reply:
So that's exactly I was expecting in Readme, now see it